### PR TITLE
chronograf (new formula)

### PR DESCRIPTION
**Pull request by gunnaraasen**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Chronograf is a time series monitoring/visualization project which seamlessly integrates with the TICK stack ([Telegraf](https://github.com/Homebrew/homebrew-core/blob/master/Formula/telegraf.rb), [InfluxDB](https://github.com/Homebrew/homebrew-core/blob/master/Formula/influxdb.rb), and [Kapacitor](https://github.com/Homebrew/homebrew-core/blob/master/Formula/kapacitor.rb)). Currently available via [homebrew-binary](https://github.com/Homebrew/homebrew-binary/blob/master/chronograf.rb) and [homebrew-cask](https://github.com/caskroom/homebrew-cask/blob/master/Casks/chronograf.rb), it recently underwent a significant rewrite and was [open-sourced](https://www.influxdata.com/announcing-the-new-chronograf-a-ui-for-the-tick-stack-and-a-complete-open-source-monitoring-solution/).

Now that Chronograf can be built directly from the source, I'd like to move the formula to homebrew-core. I've opened PRs on the other Taps to remove and redirect those formulas to homebrew-core, ([homebrew-binary PR](https://github.com/Homebrew/homebrew-binary/pull/356) and [homebrew-cask PR](https://github.com/caskroom/homebrew-cask/pull/27693)).

Thanks for the review! Let me know if there are any questions.

### CODE REVIEW

File: Formula/chronograf.rb [hunk 1]
```diff
@@ -0,0 +1,102 @@
+require "language/node"
+
+class Chronograf < Formula
+  desc "Open source monitoring and visualization UI for the TICK stack"
+  homepage "https://www.influxdata.com/time-series-platform/chronograf/"
+  url "https://github.com/influxdata/chronograf.git",
+      :tag => "1.1.0-beta2",
```
Comment (bfontaine) [inline Formula/chronograf.rb]: We need a stable release here.

Comment (gunnaraasen) [inline Formula/chronograf.rb]: @bfontaine unfortunately it looks like the "stable" version of Chronograf probably won't be released until January. I'm happy to close this PR and reopen once that has been released. Any feedback on how to fix the npm error in the tests or other general feedback would be welcome.

Comment (MikeMcQuaid) [inline Formula/chronograf.rb]: Fine to wait on this. Do you remember the test failure? Additionally, does it reproduce locally with `brew install` and `brew test`.

Comment (MikeMcQuaid) [inline Formula/chronograf.rb]: @gunnaraasen Any news on this? Thanks!

**Comment by MikeMcQuaid** [comment]
I'm now happy to accept a beta release of this just to get it out of homebrew/binary.

**Comment by MikeMcQuaid** [comment]
Tried to get this building and failed. Giving up until there's a new PR with green CI.

**Pull request closed without merging**